msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -0,0 +1,462 @@
+---
+title: Create API based Message extension
+author: v-ypalikila
+description: Learn how to build an API message extension and API plugin using Teams developer portal and Teams Toolkit.
+ms.localizationpriority: medium
+ms.topic: overview
+ms.author: anclear
+ms.date: 09/07/2023
+---
+
+# API based Message extension
+
+> [!NOTE]
+> API based message extension only supports search commands.
```

Comment:
Should we also add a public preview disclaimer here?

Reply:
Updated public preview link.